### DISCUSSION TITLE
Does @statelyai/inspect work?

### DISCUSSION THREAD

**Discussion by 7patricia** [statelyai/xstate#4945] [General]
Trying to use @statelyai/inspect but it just opens an unresponsive page every time, I can't even open dev tools to see what might be the issue. 

<img width="1728" alt="Screenshot 2024-06-17 at 15 48 52" src="https://github.com/statelyai/xstate/assets/4881189/1c679b63-18e8-4f9e-8c07-6623dcdb9401">

Any tips on how to debug this? Are other people able to use it?

**Comment by davidkpiano** [comment]
Hey Patricia, we'll investigate this. Do you have anything in context/events that may be problematic, e.g. a large payload or non-serializable objects?
